src/argilla/server/alembic/versions/bda6fe24314e_create_vectors_table.py

```diff
@@ -34,14 +34,17 @@ def upgrade() -> None:
     op.create_table(
         "vectors",
         sa.Column("value", sa.JSON(), nullable=False),
-        sa.Column("vector_settings_id", sa.Uuid(), nullable=False),
+        sa.Column("dataset_id", sa.Uuid(), nullable=False),
         sa.Column("record_id", sa.Uuid(), nullable=False),
+        sa.Column("vector_settings_id", sa.Uuid(), nullable=False),
```

Comment:
I saw this also before on PR made by Gabriel. I prefer to think in vectors like not associated with the settings in any way. But I understand that this adds flexibility to rename vectors and other stuff.